registry-requirements.md

```diff
@@ -122,3 +69,48 @@ Same content as existing binding
 (Think about how to manage the set of semantic artifacts/ontologies)
 
 Create a repository template to make it easy for people to create new bindings
+
+## Entry Requirements
+
+
+## Basic Requirements
+
+To choose a mechanism and the rules, we document our expectations below:
+
+- A binding SHOULD be written by people with a good understanding of the protocol, media type (or similar), and not necessarily the TD Editors. This includes people and organizations inside and outside of the WoT WG.
+  - Reasons Behind the Requirement:
+    - Instead of WG learning each new protocol and media type, it is more efficient for people with a good understanding of the protocol or media type to write a binding.
+    - Engaging other communities.
+- A binding SHOULD correspond to specific TD specification version(s).
+  - Reason Behind the Requirement: A binding may not fit newer or older versions of a TD specification (e.g., `readproperty` can become `readprop`, or a new operation can arrive). Thus, at the time of writing a binding, it needs to be associated with one or more known TD specification versions.
+- The binding registry MUST be a separate document but associated with a TD version.
+  - Reasons Behind the Requirement: It is easier to update in the long term.
+- Association of a binding with the TD specification (registry entry) SHOULD be confirmed by the WoT Working Group. In other words, a person needs some permission and/or confirmation to authoritatively say that a given binding can be used with TD version X. The custodian of this registry is the WoT WG.
+  - Reasons Behind the Requirement: WoT WG is the manager of the registry.
+- If the WoT WG no longer exists, the W3C Team or their delegated entity becomes the custodian.
+  - Reasons Behind the Requirement: It should be possible to maintain the registry without the WoT WG.
+- It SHOULD be possible to register, update, or remove a registry entry for associating a binding to a specific TD specification outside of the REC lifecycle of the given TD specification.
+  - Reasons Behind the Requirement:
+    - Allowing new protocols to be bound to any TD version. 
+    - The process should outlive the WG if the WG ever comes to stop functioning or if the WG decides to not work on a TD specification version anymore.
+- There SHOULD NOT be two bindings for the same protocol, media type etc.
+  - Reasons Behind the Requirement:
+    - It breaks interoperability otherwise
+    - We should avoid confusing users, i.e. which binding should I use?
+    - It would complicate implementations
```

Comment:
```suggestion
    - The process should outlive the WG, in case the WG ever dissolves or the WG decides to stop maintaining a given TD specification version.
- There SHOULD NOT be two bindings for the same protocol, media type, etc.
  - Reasons Behind the Requirement:
    - Such multiple bindings would break interoperability
    - Such multiple bindings would confuse users, i.e., which binding should I use?
    - Such multiple bindings would complicate implementations
```